app/controllers/case_assignments_controller.rb

```diff
@@ -27,20 +28,18 @@ def create
   end
 
   def destroy
-    case_assignment = CaseAssignment.find(params[:id])
-    case_assignment.destroy
+    @case_assignment.destroy
```

Comment:
Note for the future: someday we will want to not destroy case assignments but keep them as historical record. (Although we can maybe also get that out of papertrail?)